### PR TITLE
Adding the image to the front page

### PR DESCRIPTION
**Pull request by tomhamiltonstubber**

### CODE REVIEW

**Comment by sarmadkhaliq** [comment]
Footer improvement suggestion:
image

File: _includes/front/pricing.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <div class="text-center">
       <h2 class="top-pad">Pricing</h2>
-      <p>All tiers include a 14 day free trial</p>
```
Comment (tomhamiltonstubber) [inline _includes/front/pricing.html]: take the hyphen out

File: _includes/footer.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
-<section id="footer" class="bg-primary top-margin-40">
+  <section id="footer" class="bg-primary top-margin-40">
```
Comment (tomhamiltonstubber) [inline _includes/footer.html]: indent

File: lextab.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# pycparser.lextab.py. This file automatically created by PLY (version 3.4). Don't edit!
```
Comment (tomhamiltonstubber) [inline lextab.py]: Remove.

**Pull request merged**